ExtremeValueProject/DegenerateCDF.lean

```diff
@@ -116,7 +116,33 @@ lemma diracProba_is_degenerate (x₀ : ℝ) :
 at some point x₀. -/
 lemma eq_diracProba_of_isDegenerate (μ : ProbabilityMeasure ℝ) (degen : μ.cdf.IsDegenerate) :
     ∃ x₀, μ = diracProba x₀ := by
-  sorry -- **Issue #12**
+  rw [isDegenerate_iff] at degen
+  obtain ⟨x₀, h⟩ := degen
+  use x₀
+  ext s hs
+  rw [diracProba_toMeasure_apply, indicator, Pi.one_apply]
+  simp [ProbabilityMeasure.cdf, FiniteMeasure.cdf] at h
```

Comment:
This line is a [nonterminal simp](https://lean-lang.org/doc/reference/latest/The-Simplifier/Terminal-vs-Non-Terminal-Positions/). Mathlib standard is to avoid them (for maintainability reasons). For this project, the maintenance hazard is small, but since your proofs are otherwise excellent, I take the opportunity to comment on the more refined issues.

The standard routine way to get rid of a nonterminal simp is to use `simp?`, but in this case that produces a pretty ugly list of lemmas. For that reason, I found a slightly different, more readable way to `simp only`:

```suggestion
  simp only [show ⇑μ.cdf = (fun x ↦ (μ (Iic x)).toReal)
              by ext x ; rw [ProbabilityMeasure.cdf_apply_eq]] at h
```